backend/core/tests/test_user.py

```diff
@@ -1,6 +1,7 @@
 from django.db.models import manager
 from django.test import TestCase
-from core.tests.factories import TeamFactory, UserFactory
+from core.tests.factories import UserFactory
+from curriculum_tracking.tests.factories import TeamFactory
```

Comment:
Please don't move the factories.

Only:
- the viewset
- the tests for the viewset